src/plots/plots_maps.R

```diff
@@ -583,35 +598,68 @@ drawGriddedScattergeoTrace <- function(
 
 .mapUsageLeafletPlottingFunction <- function(plot) {
   zoomLevel <- .getLeafletMapZoomLevel(plot$data)
-  pallete <- colorFactor(
-    c("green", "blue", "black", "grey", "magenta3", "red"),
-    domain=c("Active", "Active(2)", "Blacklisted", "NA", "Passive", "Rejected")
-  )
 
-  leafletMap <- leaflet(data=plot$data) %>%
-    addProviderTiles(
-      "Esri.WorldStreetMap",
-      options=providerTileOptions(opacity=0.7)
-    ) %>%
-    addCircleMarkers(
-      lng=~longitude,
-      lat=~latitude,
-      popup=.getLeafletPopupContents(plot$data),
-      radius=8,
-      stroke=FALSE,
-      fillColor=~pallete(status),
-      fillOpacity=.5,
-      weight=1,
-      opacity=1,
-      color="black",
-      clusterOptions=markerClusterOptions(disableClusteringAtZoom=zoomLevel)
-    ) %>%
-    addLegend(
-      "topright",
-      pal=pallete,
-      values=~status,
-      opacity=1
-    )
+  colors = c(
+  "Active"="green", "Active(2)"="green",
+  "Rejected"="red", "Passive"="magenta3",
+  "Blacklisted"="black", "NA"="grey",
+  "Rejected in TITAN"="purple", "Domain"="cadetblue",
+  "TITAN: blacklisted"="black", "TITAN: nometa"="lightblue",
+  "TITAN: plaus"="lightgreen", "TITAN: clim"="darkblue",
+  "TITAN: buddy"="lightgray", "TITAN: SCT"="darkred",
+  "TITAN: DEM"="lightgray", "TITAN: isol"="blue",
+  "TITAN: fg"="orange", "Fraction"="beige"
+  )
+  domains = c("Active", "Active(2)", "Blacklisted", "NA",   "Rejected", "Passive", "Rejected in TITAN", "Domain",    "TITAN: blacklisted", "TITAN: nometa", "TITAN: plaus", "TITAN: clim", "TITAN: buddy", "TITAN: SCT", "TITAN: DEM", "TITAN: isol", "TITAN: fg", "Fraction")
+  pallete = colorFactor(colors, domain=domains)
+
+   if ( F ) {
```

Comment:
Hi @trygveasp ,

I'm going through your code now and wonder why the `if ( F )` here. Unless this is some feature of the R-language I'm now aware of, the code in this block is never going to be executed.

Could you give me some context for why you've chosen this?

Thanks a lot in advance.